docs/deploy-full-rp-service-in-dev.md

```diff
@@ -0,0 +1,108 @@
+# Deploy an Entire RP Development Service
+
+## Prerequisites
+
+1. Your development environment is prepared according to the steps outlined in [Prepare Your Dev Environment](./prepare-your-dev-environment.md)
+
+
+## Deploying an int-like Development RP
+
+1. Fetch the most up-to-date secrets with `make secrets`
+
+1. Copy and source your environment file.
+    ```bash
+    cp env.example env
+    vi env
+    . ./env
+    ```
+
+1. Generate the development RP configuration
+    ```bash
+    go run ./hack/genconfig > ./dev-config.yaml
+    ```
+
+1. Set environment variables based on the generated configuration.
+    ```bash
+    export KEYVAULT_PREFIX=FILL_IN   # rps[0].configuration.keyvaultPrefix
+    export DST_ACR_NAME=FILL_IN      # the prefix on '.azurecr.io' named 'rpImagePrefix' in dev-config.yaml
+    export RP_RESOURCE_GROUP=FILL_IN # rps[0].rpResourceGroupName
```

Comment:
I have also had to set 
```
export DATABASE_ACCOUNT_NAME=FILL_IN
export DATABASE_NAME=ARO
```

for hack/db / hack/ssh-agent.sh / make admin.kubeconfig to work.  FWIW I did the above by making a new env file that looks like this:

```
export LOCATION=eastus
export ARO_IMAGE=arointsvc.azurecr.io/aro:latest

. secrets/env

export RESOURCEGROUP=$USER-aro-eastus
export DATABASE_ACCOUNT_NAME=$USER-aro-eastus
export DATABASE_NAME=ARO
export KEYVAULT_PREFIX=$USER-aro-eastus
```